src/lib/setupSentry.ts

```diff
@@ -11,11 +29,11 @@ export function setupSentry(sd) {
     beforeSend(event, hint) {
       const error = hint.originalException
       if (error && error.message) {
-        if (error.message.match(/pktAnnotationHighlighter/i)) {
-          return null
```

Comment:
We may need to still `return null` here?  According to [the docs](https://docs.sentry.io/platforms/javascript/configuration/filtering/#using-beforesend) "If you return null, the event will be discarded." I think this will return `undefined`?

Reply:
Addressed by https://github.com/artsy/force/pull/6901